src/Illuminate/Encryption/OpenSSLEncrypter.php

```diff
@@ -2,13 +2,16 @@
 
 namespace Illuminate\Encryption;
 
-use RuntimeException;
 use Illuminate\Contracts\Encryption\DecryptException;
 use Illuminate\Contracts\Encryption\EncryptException;
 use Illuminate\Contracts\Encryption\Encrypter as EncrypterContract;
 
-class Encrypter implements EncrypterContract
+class OpenSSLEncrypter implements EncrypterContract
```

Comment:
IMO, it should be `OpenSsl` instead of `OpenSSL`.

Reply:
Good point, I'll change this